magellan-sample-advanced/src/main/res/layout/dog_breed.xml

```diff
@@ -4,6 +4,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
+    android:background="@android:color/white"
```

Comment:
For GPU overdraw reasons, we should encourage users to keep their `Step`'s views transparent. Could we automatically add an `?colorBackground`-colored background to `ShowTransition` and other transitions that overlap?

Reply:
Not sure about this.